contrib/docker/Dockerfile

```diff
@@ -80,6 +80,7 @@ EXPOSE 9108 9109 19108 19109 18555 19556
 ENTRYPOINT [ "/bin/entrypoint" ]
 
 RUN [ "dcrd", "--version" ]
+RUN [ "dcrctl", "--version" ]
 
```

Comment:
Will this ever run? I think docker will stop at `ENTRYPOINT [ "/bin/entrypoint" ]` when executing this image.

Reply:
It runs. Why would docker stop at `ENTRYPOINT`?

Reply:
`/bin/entrypoint` is executed when I run the image.

Reply:
Those commands are not executed when the image is run, rather they are executed as part of building the image. You will only see the output of them at build time. Note that you will only see **the first time** the build is run because from that point onwards the layer will be cached.

Reply:
> Those commands are not executed when the image is run, rather they are executed as part of building the image. You will only see the output of them at build time. Note that you will only see the output **the first time** the build is run because from that point onwards the layer will be cached.

Ah, I see. Thanks for the explanation. I've built the image several times and I must have missed the initial output.